src/client.js

```diff
@@ -203,16 +198,16 @@ module.exports = class IrcClient extends EventEmitter {
             // Add a reply() function to selected message events
             if (['privmsg', 'notice', 'action'].indexOf(event_name) > -1) {
                 event_arg.reply = function(message) {
-                    var dest = event_arg.target === client.user.nick ?
-                        event_arg.nick :
-                        event_arg.target;
+                    var dest = event_arg.target === client.user.nick
```

Comment:
Ternary ops should use the existing format in other Kiwi projects:
~~~javascript
foo = somevar === true ?
    'truthy' :
    'falsy';
~~~

Reply:
Changed.

https://github.com/kiwiirc/irc-framework/compare/d0d223c567c19334cca0cdc71eb7f6966fe110a2..6b8671a311e9e183efe2caa219ad01f71b0b41f4